### PR TITLE
added support for stop_sequences params on Vertex AI

### PR DESCRIPTION
**Pull request by lkuligin**
Added a native support for stop words (`stop_sequences`), now exposed by Vertex AI:
https://github.com/googleapis/python-aiplatform/releases/tag/v1.31.0

### CODE REVIEW

File: libs/langchain/langchain/llms/vertexai.py [hunk 1]
```diff
@@ -139,8 +131,9 @@ def _predict(
         self, prompt: str, stop: Optional[List[str]] = None, **kwargs: Any
     ) -> str:
         params = {**self._default_params, **kwargs}
-        res = completion_with_retry(self, prompt, **params)  # type: ignore
-        return self._enforce_stop_words(res.text, stop)
+        if stop:
+            params["stop_sequences"] = stop
```
Comment (baskaryan) [inline libs/langchain/langchain/llms/vertexai.py:135]: should we also check if self.stop was specified?

Comment (lkuligin) [reply libs/langchain/langchain/llms/vertexai.py]: Done!

**Comment by lkuligin** [comment]
@baskaryan could you give it another look, please?

**Comment by lkuligin** [comment]
@baskaryan a friendly ping on this, please

**Comment by lkuligin** [comment]
@baskaryan @eyurtsev this PR has been waiting for a while, can you take a look, please?

**Comment by lkuligin** [comment]
@eyurtsev @baskaryan @hwchase17 folks, I'm sorry for pinging you again, and I appreciate you're busy, but could you merge it, please?

**Comment by baskaryan** [comment]
@lkuligin think we added in another pr, ok to close?

**Comment by lkuligin** [comment]
yep, sure, let's close this one!

**Pull request closed without merging**